sdks/rust/build.rs

```diff
@@ -4,7 +4,7 @@ fn main() {
         .build_server(false)
         .compile(
             &["proto/sdk/alpha/alpha.proto", "proto/sdk/sdk.proto"],
-            &["proto/googleapis", "proto/sdk/alpha", "proto/sdk"],
+            &["proto/googleapis", "proto/grpc-gateway", "proto/sdk/alpha", "proto/sdk"],
```

Comment:
Add build path, but rust sdk use `tonic` instread of `protoc`. So there are not many changes due to this PR
https://github.com/googleforgames/agones/pull/2112